apps/system/index.html

```diff
@@ -280,9 +282,11 @@
                 <!-- App install manager notification -->
                 <div id="install-manager-notification-container">
                 </div>
+<!--IFNDEF_NO_BLUETOOTH
                 <!-- Bluetooth transfer -->
```

Comment:
Hmm, indeed, I was looking at an example elsewhere and chose to use this syntax (https://github.com/mozilla-b2g/gaia/blob/cec4c1d3729137a24163756d15f98b0d37803966/apps/settings/elements/root.html#L140) - but looking at the regex in `build/preprocessor.js` it seems a fully formed comment should work, and in fact, it appears to work for me.